app/models/user_score.rb

```diff
@@ -0,0 +1,17 @@
+class UserScore < ApplicationRecord
+  has_many :subscores
+  belongs_to :user
+
+  def get_total_score
+    score = 0
+    max_score = 0
+    self.subscores.each do |subscore|
+      score += subscore.score
+      max_score += subscore.max_score
+    end
+    {
```

Comment:
You could add a return here, but I really don't care either way. Just looked a little funny to me